doc/nrf/glossary.rst

```diff
@@ -306,36 +331,47 @@ Glossary
    Over-the-Air (OTA)
       Any type of wireless transmission.
 
+   Packet Traffic Arbitration (PTA)
+      A collaborative coexistence mechanism for colocated wireless protocols.
+
    Patch
       A method to describe changes in one or more source code files.
       It does not require a repository.
-      Sometimes it is improperly used as a synonym of commit.
+      Sometimes it is improperly used as a synonym of :term:`commit<Commit>`.
 
    Power Amplifier (PA)
       A device used to increase the transmit power level of a radio signal.
 
    Power Saving Mode (PSM)
-      A feature introduced in 3GPP Release 12 to improve battery life of IoT (Internet of Things) devices by minimizing energy consumption.
+      A feature introduced in 3GPP Release 12 to improve the battery life of IoT (Internet of Things) devices by minimizing energy consumption.
       The device stays dormant during the PSM window.
 
    Printed Circuit Board (PCB)
       A board that connects electronic components.
 
+   Programmable Peripheral Interconnect (PPI)
+      It enables peripherals to interact autonomously with each other using tasks and events independently of the CPU.
+
    Protocol Data Unit (PDU)
       Information transferred as a single unit between peer entities of a computer network and containing control and address information or data.
       PDU mode is one of the two ways of sending and receiving SMS messages.
 
    Provisioning
-      * In a thread mesh network, the process of associating a device to the appropriate service, and performing any application or vendor-specific configuration.
+      * In a Thread Mesh network, the process of associating a device to the appropriate service, and performing any application or vendor-specific configuration.
         It is a step in the commissioning process.
-      * In a bluetooth mesh network, the process of adding devices to the network.
+      * In a Bluetooth Mesh network, the process of adding devices to the network.
       * In a bootloader, the process of storing public key hashes in a separate region of memory from the bootloader image.
 
    Pull Request
       A set of commits that are sent to contribute to a repository.
 
+   Qualified Design Identification (QDID)
+      A unique identifier assigned to a design that has completed the Bluetooth Qualification Process.
+
    Quality of Service (QoS)
-      The measured overall performance of a service, such as a telephony or a computer network, or a cloud computing service.
+      * The measured overall performance of a service, such as a telephony or a computer network, or a cloud computing service.
+      * A module in the SoftDevice Controller dedicated to providing these signal quality measurements.
```

Comment:
I am not sure this is accurate. @rugeGerritsen ?

Reply:
I would suggest to leave out "A module in the SoftDevice Controller " and rather rewrite the definition of QoS to something similar to:
```suggestion
      * The measured overall performance of a service, such as a network, connection, or a cloud computing service.
```

Reply:
fixed